CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Fixed
+- Parameter and objective classes are now slotted due to `cached_property` attrs support
```

Comment:
@AdrianSosic what is preventing us from rebasing/merging this? Is there anything still open?

Reply:
Will show you in a minute by linking to the corresponding issue (and closing this thread). Will need to put on hold again until resolved =/